application/libraries/admin/pdf.php

```diff
@@ -843,7 +843,7 @@ public function write_out($name)
 
     public function delete_html($text)
     {
-        $text = html_entity_decode((string) $text, null, 'UTF-8');
+        $text = html_entity_decode((string) $text, ENT_QUOTES|ENT_SUBSTITUTE, 'UTF-8');
```

Comment:
Moving from `ENT_QUOTES | ENT_SUBSTITUTE | ENT_HTML401` to `ENT_QUOTES | ENT_SUBSTITUTE` no ?

Seems it's OK, but it update something.